config-builder/src/test/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/JmsDestinationEntityBuilderTest.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.bundle.builder;
+
+public class JmsDestinationEntityBuilderTest {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add some tests to this class. ([squid:S2187](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2187))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=138&issues=AWc81LjHfimTzgTvo62F&open=AWc81LjHfimTzgTvo62F)

[ISSUE_KEY]: AWc81LjHfimTzgTvo62F